obspy/core/tests/test_stream.py

```diff
@@ -2011,18 +2011,22 @@ def test_rotate(self):
         st = read()
         st += st.copy()
         st[3:].normalize()
+        for tr in st[3:]:
+            tr.stats.location = '01'
```

Comment:
Tbh, I think it's a weird test case, why would you process a fully duplicated exact same waveform set of 2x the same three Traces?

Reply:
Yes, that is completely nonsense. For me, it conceptually represents the rotation of waveforms of two different events (with different baz) registered at the same station. Changing the starttime instead of the location code would be more in line with this idea. I do not know if this was the original intention.